build.gradle

```diff
@@ -39,6 +39,7 @@ buildscript {
         classpath 'com.google.gms:google-services:4.3.15'
         classpath "com.ncorti.ktfmt.gradle:plugin:0.11.0"
         classpath 'com.google.firebase:firebase-appdistribution-gradle:3.0.2'
+        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.9.5'
```

Comment:
Why is this change in here? Is this relevant to this integration?

Reply:
Yes, otherwise any test app in this repo can't have Crashlytics.